docs/kyma/14-04-common-errors.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Common installation errors
+type: Troubleshooting
+---
+
+## Istio-related error
+
+In some cases, the logs of Kyma installer may show this error, which seemingly indicates problems with Istio:
+
+```
+Step error:  Details: Helm install error: rpc error: code = Unknown desc = validation failed: [unable to recognize "": no matches for kind "DestinationRule" in version "networking.istio.io/v1alpha3", unable to recognize "": no matches for kind "DestinationRule" in version "networking.istio.io/v1alpha3", unable to recognize "": no matches for kind "attributemanifest" in version "config.istio.io/v1alpha2"
+```
+
+As Istio is the first sizeable component handled by the Kyma Installer, sometimes not all of the required CRDs are created before the Installer proceeds to the next component. This situation doesn't cause the installation to fail.
+Instead, the Istio installation step repeats and gets more time for setup and the error message is logged regardless of that.
+
+## Job failed: DeadlineExceeded error
+
+The `Job failed: DeadlineExceeded` error indicates that a job object didn't finish in a set time leading to a timeout. Frequently this error is followed by a message that indicates the release which failed to install: `Helm install error: rpc error: code = Unknown desc = a release named core already exists`.
+
+As this error is caused by a timeout, restart the installation.
```

Comment:
time out, time-out: Always hyphenate as an adjective or a noun. Don't hyphenate as a verb phrase, such as the connection timed out. [source: [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/time-out)] :P